stratum/hal/lib/common/gnmi_publisher.cc

```diff
@@ -34,6 +34,13 @@ ::util::Status GnmiPublisher::HandleUpdate(
     CopyOnWriteChassisConfig* config) {
   absl::WriterMutexLock l(&access_lock_);
 
+  // Check for IPsec use-case
```

Comment:
"Check for IPsec use case" is a little vague.

Looking at this piece of code in isolation, I don't understand what we're testing, what we're doing as a result of it, or why we're doing it.

The comment is kinda like "see da guy about da t'ing" -- "check for the, you know, and do the thing that comes afterwards, for, you know, IPsec."

The comment is for the benefit of someone who _doesn't_ already know what you're talking about.